3.2/itop_design.xsd

```diff
@@ -2804,15 +2811,30 @@
             <xs:extension base="MenuBase">
                 <xs:all>
                     <xs:element name="rank" type="xs:decimal" minOccurs="0" />
-                    <xs:element name="auto_reload" type="xs:string" minOccurs="0" />
-                    <xs:element name="enable_admin_only" type="xs:byte" minOccurs="0" />
+                    <xs:element name="auto_reload" minOccurs="0" default="none">
+                        <xs:simpleType>
+                            <xs:union>
+                                <xs:simpleType>
+                                    <xs:restriction base="xs:string">
+                                        <xs:enumeration value="none" />
+                                        <xs:enumeration value="standard" />
+                                        <xs:enumeration value="fast" />
+                                    </xs:restriction>
+                                </xs:simpleType>
+                                <xs:simpleType>
+                                    <xs:restriction base="xs:nonNegativeInteger" />
+                                </xs:simpleType>
+                            </xs:union>
+                        </xs:simpleType>
+                    </xs:element>
+                    <xs:element name="enable_admin_only" type="zeroOrOneEnumeration" minOccurs="0" />
                     <xs:element name="parent" type="xs:string" minOccurs="0" />
                     <xs:element name="oql" type="xs:string" minOccurs="0" />
-                    <xs:element name="do_search" type="xs:string" minOccurs="0" />
-                    <xs:element name="search_form_open" type="xs:string" minOccurs="0" />
+                    <xs:element name="do_search" type="zeroOrOneEnumeration" minOccurs="0" />
```

Comment:
Combodo always uses `1` or an empty value for `do_search`, I did as well.

Should we allow empty values or do we want to restrict this even against use case of Combodo itself?